### PR TITLE
Design reorder columns

### PR DESCRIPTION
**Pull request by lilyclements**
@africanmathsinitiative/developers this is ready to review

### CODE REVIEW

File: instat/ucrReorder.vb [hunk 1]
```diff
@@ -227,7 +227,7 @@ Public Class ucrReorder
     End Sub
 
     'to update this to check if the order has changed
-    Public Function isEmpty() As Boolean
+    Public Function IsEmpty() As Boolean
```
Comment (lilyclements) [inline instat/ucrReorder.vb]: Naming convention for consistency, this means a minor edit in dlgReorderDataFrame where it updated this function name

**Comment by lilyclements** [comment]
@africanmathsinitiative/developers this is still waiting for review. It should be a quick review

**Comment by lilyclements** [comment]
@africanmathsinitiative/developers I need someone to review this as soon as possible for reasons outlined in #3119

**Comment by lilyclements** [comment]
@africanmathsinitiative/developers this is waiting for review.

**Comment by shadrackkibet** [comment]
I am going to test this now.

**Comment by shadrackkibet** [comment]
I have tested this and it is working fine.@dannyparsons this ready for you to review.

**Pull request merged**